indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -73,38 +73,56 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
     renderDividers() {
         const nDays = this.dates.length;
         const daySize = (100 / nDays);
-
         const {dateRange} = this.props;
+
         const emptyDays = this.dates
             .filter(day => dateRange.length !== 0 && !dateRange.includes(day))
             .map(day => this.dates.findIndex((el) => el === day));
 
+        const weekendDays = this.dates
```

Comment:
I noticed you have this same operation in `renderHeader()`, perhaps could this be a separate function? It even could be a util function (`date.js`).

Apart from that, I think the weekend days aren't being properly calculated. For instance for the March you're highlighting Sunday and Monday.
![image](https://user-images.githubusercontent.com/8863238/54594315-ad6d0700-4a30-11e9-96dc-17df14c540a4.png)